Segfault from panaceacli when deleting DIDs

```
panic: invalid character 'd' looking for beginning of value

goroutine 1 [running]:
github.com/cosmos/cosmos-sdk/types.MustSortJSON(...)
        /go/pkg/mod/github.com/medibloc/cosmos-sdk@v0.35.3/types/utils.go:40
github.com/medibloc/panacea-core/x/did/types.MustGetSignBytesWithSeq(0xc0004fa5b0, 0x13d5b00, 0xc000012000, 0x2, 0x0, 0x0, 0x13d4f20)
        /src/x/did/types/auth.go:27 +0x13b
github.com/medibloc/panacea-core/x/did/client/cli.sign(0xc0004fa5b0, 0xc000012810, 0x13fc4e0, 0xc000c49ec0, 0x0, 0x0, 0x13d4f20, 0xc000010018, 0xf2fdbe, 0x4, ...)
        /src/x/did/client/cli/tx.go:288 +0x14c
github.com/medibloc/panacea-core/x/did/client/cli.GetCmdDeleteDID.func1(0xc000c68f00, 0xc000c7a1c0, 0x2, 0x4, 0x0, 0x0)
        /src/x/did/client/cli/tx.go:158 +0x347
github.com/spf13/cobra.(*Command).execute(0xc000c68f00, 0xc000c7a140, 0x4, 0x4, 0xc000c68f00, 0xc000c7a140)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000c2e280, 0x8bd765, 0xc000c2e280, 0xf2ee38)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x30b
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/tendermint/tendermint/libs/cli.Executor.Execute(0xc000c2e280, 0x12fc278, 0x2, 0xc000039100)
        /go/pkg/mod/github.com/tendermint/tendermint@v0.31.9/libs/cli/setup.go:89 +0x3c
main.main()
        /src/cmd/panaceacli/main.go:121 +0x911
```

--- a/x/did/types/did.go
+++ b/x/did/types/did.go
@@ -48,7 +48,7 @@ func (did DID) Empty() bool {
 
 // GetSignBytes returns a byte array which is used to generate a signature for verifying DID ownership.
 func (did DID) GetSignBytes() []byte {
-	return []byte(did)
+	return sdk.MustSortJSON(didCodec.MustMarshalJSON(did))
 }
 
 type NetworkID string

--- a/x/did/types/did_test.go
+++ b/x/did/types/did_test.go
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/cosmos/cosmos-sdk/codec"
+
 	"github.com/btcsuite/btcutil/base58"
 
 	"github.com/stretchr/testify/require"
@@ -33,6 +35,13 @@ func TestNewDIDFrom(t *testing.T) {
 func TestDID_Empty(t *testing.T) {
 	require.True(t, DID("").Empty())
 	require.False(t, DID("did:panacea:testnet:KS5zGZt66Me8MCctZBYrP").Empty())
+}
+
+func TestDID_GetSignBytes(t *testing.T) {
+	did := DID("did:panacea:testnet:KS5zGZt66Me8MCctZBYrP")
+	var did2 DID
+	require.NoError(t, codec.New().UnmarshalJSON(did.GetSignBytes(), &did2))
+	require.Equal(t, did, did2)
 }
 
 func TestNewNetworkID(t *testing.T) {

